reflectapi-schema/src/lib.rs

```diff
@@ -938,6 +938,14 @@ impl Field {
         self.name.as_str()
     }
 
+    pub fn is_named(&self) -> bool {
+        !self.is_unnamed()
+    }
+
+    pub fn is_unnamed(&self) -> bool {
+        self.name.parse::<u64>().is_ok()
```

Comment:
you are certain we do not miss serde_name usage here? just curious why these were needed for names but not for serde names

Reply:
Didn't think too hard about it. This is copied from `rust/ts` codegen. I think the clearest solution would be to make `name: Option<String>` rather than checking if it's a number or not.

Reply:
I see what you mean here. It is fine that names are numbers. It is by design. It captures full intent. Codegen can interpret it as it needs to.